src/tests/tests.yaml

```diff
@@ -1,9 +1,23 @@
 charm_name: cinder-netapp
 tests:
-  - tests.tests_cinder_netapp.CindernetappTest
+  - zaza.openstack.charm_tests.cinder_netapp.tests.CinderNetAppTest
 configure:
   - zaza.openstack.charm_tests.keystone.setup.add_demo_user
 gate_bundles:
-  - xenial-stein
+  - bionic-queens
+  - bionic-rocky
+  - bionic-stein
+  - bionic-train
+  - bionic-ussuri
+  - focal-ussuri
+  - focal-victoria
+  - focal-wallaby
+  - groovy-victoria
+  - hirsute-wallaby
 smoke_bundles:
-  - xenial-stein
+  - bionic-ussuri
+dev_bundles:
+  - impish-xena
```

Comment:
Should we add `focal-xena` to this list as well?

Reply:
Yep, we should.